filter.lua

```diff
@@ -79,25 +79,54 @@ local function match(lists, domain, action)
 	end
 end
 
-local function filter(packet, offset, dport, allow, deny, outbox)
+local function mac_ntoa(mac)
+	local ret = {}
+	for i = 1, 6 do
+		table.insert(ret, string.format("%02x", string.byte(mac, i)))
+	end
+	return table.concat(ret, ":")
+end
+
+local function inet_ntoa(addr)
```

Comment:
If I'm not mistaken, we have a function like that in lunatik/socket/inet, can't we use that? If it's the case, we could move it out from socket.. however, we can do it later as well.. I would just open a ticket to tackle it later if it's the case

Reply:
fair enough. we could indeed give it some more thought, and maybe reactor around a bit further as you suggested. I'm pretty sure there are a couple more things in dome that we could (maybe?) move a separate module in lunatik?

anyways, we could remove this function now, use Lunatik's `inet.ntoa` as-is, and create a ticket to organize it down the line. what do you think?

Reply:
sorry, lunatik won't let me `require("socket.inet")` from softirq. I'll open the ticket.